src/main/java/com/shanebeestudios/mcdeop/Version.java

```diff
@@ -32,9 +32,17 @@
     SERVER_1_17_1(Type.SERVER, "1.17.1", "a16d67e5807f57fc4e550299cf20226194497dc2", "f6cae1c5c1255f68ba4834b16a0da6a09621fe13"),
     CLIENT_1_17_1(Type.CLIENT, "1.17.1", "8d9b65467c7913fcf6f5b2e729d44a1e00fde150", "e4d540e0cba05a6097e885dffdf363e621f87d3f"),
 
-    // Snapshot
+    // Snapshots for 1.18
```

Comment:
This doesn't need to be changed.
This is just an internal note for keeping these sections clean. I dont want to have to change the note each version and cause unnecessary line changes.